website/docs/reference/resource-configs/postgres-configs.md

```diff
@@ -148,4 +148,24 @@ models:
 ```
 </File>
 
+### Limitations
+
```

Comment:
We don't generally like to have two headers back-to-back. Is there a short sentence or two we can add here that would sprinkle in some commentary?

Reply:
```suggestion

Below are current limitations that we hope to address in a future release.
```